src/index.css

```diff
@@ -121,6 +121,16 @@ body {
   }
 }
 
+.badgeBlockName {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
```

Comment:
The docs say, that you can use `flex-direction`: https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient

Reply:
Set plugin instead